lib/runtimeSettings.js

```diff
@@ -27,6 +32,22 @@ function getSettingsFile (settings) {
         if (settings.settings.codeEditor) {
             projectSettings.codeEditor = settings.settings.codeEditor
         }
+        if (settings.settings.theme) {
+            projectSettings.theme = `theme: '${settings.settings.theme}',`
+        }
+        if (settings.settings.page?.title) {
+            projectSettings.page_title = settings.settings.page.title
+        }
+        projectSettings.page_favicon = 'favicon: "/theme/css/favicon.ico",' // TODO: get from theme
```

Comment:
I appreciate the TODO here, but would it be better to only set favicon *if* the settings object has specified a theme? Not sure what this will do if there's no theme at all.

Reply:
Yeah. I'll revisit this.

Reply:
Happy to leave as a todo for now. There's plenty of scope for additional theming options and this is one of them